scripts/generateAssetData/utils/portals.ts

```diff
@@ -131,8 +131,8 @@ export const getPortalTokens = async (nativeAsset: Asset): Promise<Asset[]> => {
     return {
       ...explorerData,
       color: colorMap[assetId] ?? '#FFFFFF',
-      icon: token.images?.[0] ?? '',
-      icons: token.images ?? [],
+      icon: token.images?.[0],
+      icons: token.images,
```

Comment:
Doesn't generate `icons` array in `generatedAssetData.json` anymore - see loss of sanity in https://github.com/shapeshift/web/pull/7409/files#r1689388460